pilot/pkg/bootstrap/server_test.go

```diff
@@ -119,6 +192,17 @@ func TestNewServerCertInit(t *testing.T) {
 		t.Run(c.name, func(t *testing.T) {
 			test.SetForTest(t, &features.PilotCertProvider, c.certProvider)
 			test.SetForTest(t, &features.EnableCAServer, c.enableCA)
+
+			// check if we have some tls assets to write for test
+			if c.FSCertsPaths != (TLSFSLoadPaths{}) {
+				err := loadCertFilesAtPaths(c.FSCertsPaths)
```

Comment:
nit: we might be able to do `dir := t.TempDir; os.Chdir(dir)` then everything will be relative to our auto-cleaned up tmp dir.